tests/scripts/thread-cert/Cert_8_1_01_Commissioning.py

```diff
@@ -65,7 +65,7 @@ def test(self):
 
         self.nodes[JOINER].interface_up()
         self.nodes[JOINER].joiner_start('openthread')
-        time.sleep(10)
+        time.sleep(15)
```

Comment:
Is this delay/timeout defined by the certification test, or just an arbitrary delay? It it is the former, we shouldn't change it. If it is the later, do we fully understand why 15 seconds is more reasonable than 10?